api/v1beta2/ibmpowervscluster_webhook.go

```diff
@@ -137,19 +137,5 @@ func (r *IBMPowerVSCluster) validateIBMPowerVSClusterCreateInfraPrereq() *field.
 		return nil
```

Comment:
Please remove this check as well, Its no more needed.

Reply:
Done.